nengo_fpga/networks/fpga_pes_ensemble_network.py

```diff
@@ -422,12 +421,9 @@ def build_FpgaPesEnsembleNetwork(model, network):
             ens_args['neuron_type'] = \
                 network.neuron_str_map[type(network.ensemble.neuron_type)]
         else:
-            warn_str = \
-                'Neuron type "' + str(type(network.ensemble.neuron_type)) + \
-                '" is currently not supported. Using default "' + \
-                str(type(network.default_neuron_type)) + '" neuron.'
-            logger.warn(warn_str)
-            print('WARNING: ' + warn_str)
+            raise nengo.exceptions.BuildError(
```

Comment:
Hmm. throwing the exception seems to cause a little problem in the cleanup code (line 206). 
```
Traceback (most recent call last):
  File "/home/xchoo/git/nengo-fpga/nengo_fpga/simulator.py", line 74, in terminate
    net.cleanup()
  File "/home/xchoo/git/nengo-fpga/nengo_fpga/networks/fpga_pes_ensemble_network.py", line 206, in cleanup
    os.remove(self.local_data_filepath)
IsADirectoryError: [Errno 21] Is a directory: './'
```

Should be fixable by changing `os.path.exists` to `os.path.isfile`

Reply:
Weird I never saw this. I changed the cleanup check like oyu suggested, let me know if that fixes it.